modules/ROOT/pages/.get-mapping-recommendations.adoc

```diff
@@ -0,0 +1,86 @@
+= Get a Recommendation for Mapping Payloads for a Transformation
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+In the Transform card, you can view recommendations for how to map the data elements in an input payload to the data elements in an output payload. If a recommendation suits your needs, you can click *Apply* to save the mapping and use it in the card.
+
+== About this task
+
+The Transform card works by converting messages from the previous card in a flow to an output messages that can be consumed by the next card. A message can consist of a payload, attributes, and variables. In this image, the Transform card is placed between and HTTP Listener card and an HTTP Request card. The Transform card can accept  messages (which contain data in JSON or in some other format) from the HTTP Listener and transform them into messages that the HTTP Request card is configured to accept.
```

Comment:
...from the previous card in a flow to an output messages...

Should be either flow to output messages or flow to an output message

Reply:
In this image, the Transform card is placed between **and** HTTP Listener card

Should be "an"